akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectiveExtendedExampleSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.http.scaladsl.server.directives
+
+import akka.event.{LoggingAdapter, Logging}
+import akka.event.Logging.LogLevel
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.server.RouteResult.Complete
+import akka.http.scaladsl.server.directives.{LoggingMagnet, LogEntry, DebuggingDirectives}
+import docs.http.scaladsl.server.RoutingSpec
+import Console._
+
+//This example shows how to log client request and response time using DebuggingDirective
+
+class DebuggingDirectiveExtendedExampleSpec extends RoutingSpec {
+
+  "logRequestResultWithResponseTime" in {
+
+    val THRESHOLD_VALUE = 3
```

Comment:
I think we can skip the threshold as well, you are including the elapsed time in the log so that if requests takes more than an arbitrary limit could be seen in the logs after the fact